DSA/4.1 Stacks/01. Introduction/3.Introduction_to_Stacks_video.md

```diff
@@ -0,0 +1,15 @@
+## Introduction to stacks:
+ A stack is an Abstract Data Type (ADT), commonly used in most programming languages. It is named stack as it behaves like a real-world stack, for example – a deck of cards or a pile of plates, etc.
+
+ ## Link for the Video(click on the image):
+
+ [![image](https://i.ibb.co/6mC1LRX/Screenshot-2021-11-14-at-11-48-47-PM.png)](https://youtu.be/nwNHV98-nao)
```

Comment:
There is no need to add a video on stack that issue was already assigned or created